2-patterns_integrations/src/main/java/com/epam/dhontar/aqamp/utils/enums/ServicesEndpoints.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.dhontar.aqamp.utils.enums;
+
+public enum ServicesEndpoints {
+    //    SITE_URL("https://fakerestapi.azurewebsites.net/index.html"),
+    BASE_URL("https://fakerestapi.azurewebsites.net/api/v1/"),
+    AUTHORS_URL("Authors/"),
+    USERS_URL("Users/"),
+    SLACK_API_URL("");
```

Comment:
Empty url?

Reply:
it is empty just to avoid blocking of a webhook in slack. as soon as its key appears on GitHub it is automatically blocked